templates/drupal-7/docker-compose.server.yml

```diff
@@ -0,0 +1,69 @@
+version: "3"
+
+networks:
+  frontend:
+    external: true
+  app:
+    driver: bridge
+    internal: true
+    external: false
+
+services:
+  phpfpm:
+    image: itkdev/php7.4-fpm:alpine
```

Comment:
Not sure this is the right way long term. I know we currently don't do semver or the like on our images. But I still think this is a misuse of the tag system unless we plan to switch all images to alpine

Reply:
Update: I can see nginx doing something similar `nginx:stable-alpine`. Need to think about this.

Reply:
It's the way 90% of image om docker hub do it.